packages/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -448,7 +448,7 @@ describe('plugin-mercury', () => {
             assert.calledWith(
               Socket.prototype.open,
               sinon.match(/ws:\/\/providedurl.com/),
-              sinon.match.any
+              sinon.match.has('clientTimestamp', clock.now + 2) // 2 for 2 ticks
```

Comment:
where are the 2 ticks coming from? shouldn't we just mock the current time with `sinon.useFakeTimers()`?

Reply:
We can do that, but I'm using the clock that is defined above. The 2 ticks come from the beforeEach function above with `process.nextTick`. - was trying to utilize the code which already exists here.

Reply:
ah, ok, so yeah we are already using fake timers, it's a bit convoluted why +2, but I guess it's fine

Reply:
I've changed this to just do one tick (for the mock websocket) and using fake timers